tools/project_manager/automodify.php

```diff
@@ -217,13 +226,17 @@ function ensure_project_blurb( $project )
         $n_hours_to_wait = 4;
         $max_reclaimable_time = time() - $n_hours_to_wait * 60 * 60;
 
-        $res = mysqli_query(DPDatabase::get_connection(), "
+        $sql = sprintf("
             SELECT image
             FROM $projectid
-            WHERE state IN ('$round->page_out_state','$round->page_temp_state')
-                AND $round->time_column_name <= $max_reclaimable_time
-            ORDER BY image ASC
-        ");
+            WHERE state IN ('%s','%s')
+                AND %d <= %d
```

Comment:
We can't escape column names with `DPDatabase::escape()` . Since the column name is defined in the round, I wouldn't escape it but leave it directly into the string.